build.sbt

```diff
@@ -1,68 +1,74 @@
+import Dependencies._
+
+lazy val scala211 = "2.11.12"
```

Comment:
Why did you make these `lazy val`s?

Reply:
Mainly just following the conventions in the SBT documentation. Probably not strictly necessary here because they are used immediately after, however, an example of an unused val is the scala213 val (for now).